src/helpers/messaging.rs

```diff
@@ -252,6 +252,7 @@ impl Debug for ReceiveRequest {
 mod tests {
     use crate::ff::Fp31;
     use crate::helpers::Role;
+    use crate::protocol::context::ProtocolContext;
```

Comment:
I forgot to make this suggestion.  I think that we should rename `ProtocolContext` (and `...ProtocolContext` to `Context` (and `...Context`) and save some typing.

Reply:
I like this suggestion.